src/desktop-container-type-context.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export type ContainerType = 'small' | 'medium' | 'large';
+
+const DesktopContainerTypeContext = React.createContext<ContainerType | null>(null);
+
+export const useDesktopContainerType = (): ContainerType | null =>
+    React.useContext(DesktopContainerTypeContext);
```

Comment:
The hook and context should be exposed in the index

Reply:
no, it's an implementation detail and will only be used inside mistica for the moment.
If we need to expose it in the future, will do it

Reply:
agree, its internal by now